<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowSettingClass</class>
 <widget class="QWidget" name="MainWindowSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>软件配置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/icon.ico</normaloff>../res/icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>综合管理软件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid rgb(192, 192, 192); /* 默认边框颜色 */
}
QLineEdit:focus {
    border: 1px solid rgb(50, 170, 233); /* 焦点时的边框颜色 */
    outline: none;                       /* 移除默认虚线框 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>知识图谱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid rgb(192, 192, 192); /* 默认边框颜色 */
}
QLineEdit:focus {
    border: 1px solid rgb(50, 170, 233); /* 焦点时的边框颜色 */
    outline: none;                       /* 移除默认虚线框 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TMAX3D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid rgb(192, 192, 192); /* 默认边框颜色 */
}
QLineEdit:focus {
    border: 1px solid rgb(50, 170, 233); /* 焦点时的边框颜色 */
    outline: none;                       /* 移除默认虚线框 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
